client/src/pages/dashboard/teams/[teamId]/games/index.js

```diff
@@ -1,22 +1,136 @@
+                    stats === 'steals' &&
+                    gamesStats[i][j].steals &&
+                    (statsTotal = statsTotal + gamesStats[i][j].steals)
+            }
+        }
+
+        return gamesStats.length !== 0 ? (
+            <Td paddingLeft="1.75rem">
+                {Math.round((statsTotal / gamesStats.length) * 100) / 100}
+            </Td>
+        ) : (
+            <Td paddingLeft="1.75rem">0</Td>
+        )
+    }
+
+    const PlayersTable = ({ players, statsFields }) => (
+        <Table variant="striped" size="sm">
+            <Thead>
+                <Tr borderBottom="2px solid grey" fontWeight="semibold">
+                    <Th minW="100px">PLAYER</Th>
+                    {statsFields.map((s) => (
+                        <Th key={s}>{`AVERAGE ${s.toUpperCase()}`}</Th>
+                    ))}
+                </Tr>
+            </Thead>
+            <Tbody>
+                {players?.map((player) => {
+                    return (
+                        <Tr fontSize="1.1rem">
+                            <Td>{player.name}</Td>
+                            {statsFields.map((s) => (
+                                <Totals stats={s} playerId={player.id} gamesStats={gamesStats} />
+                            ))}
+                        </Tr>
+                    )
+                })}
+            </Tbody>
+        </Table>
+    )
 
     return (
         <Template>
             <Head>
-                <title>Dribblr | Team Details</title>
+                <title>Dribblr | {team?.team?.name || 'Team Details'}</title>
             </Head>
-            <Container>
-                <VStack spacing="1.25rem"></VStack>
+            <Container heading={heading} tag={tag} hover={hover}>
+                <VStack spacing="1.25rem" align="left">
+                    <Tabs variant="soft-rounded" colorScheme="gray" w="100%">
+                        <TabList w="100%" paddingY="0.5rem" paddingX="1rem" color="greyText.500">
+                            <Center w="100%">
+                                <Tab mr="1rem">Players</Tab>
+                                <Tab ml="1rem">Matches</Tab>
+                            </Center>
+                        </TabList>
+                        <TabPanels w="100%">
+                            <TabPanel w="100%" overflow="auto" maxHeight="400px">
+                                <PlayersTable
+                                    players={team?.team?.players}
+                                    statsFields={STATS_FIELDS}
+                                />
+                            </TabPanel>
+                            <TabPanel w="100%" overflow="auto" maxHeight="400px">
+                                <VStack spacing="0.5rem" w="100%">
+                                    {team?.team?.games.map((game) => (
+                                        <HStack key={game._id} w="100%">
+                                            <MatchContainer game={game} />
```

Comment:
I could be wrong, but `MatchContainer` component just appends `game.round}/games/game._id` to the url, so this route would be incorrect.